Marlin/stepper_indirection.h

```diff
@@ -84,6 +89,9 @@
     #if ENABLED(HAVE_TMC2130) && ENABLED(X_IS_TMC2130)
       extern TMC2130Stepper stepperX;
     #endif
+    #if ENABLED(HAVE_TMC2208) && ENABLED(X_IS_TMC2208)
```

Comment:
You may use `#elif ENABLED(HAVE_TMC2208) && ENABLED(X_IS_TMC2208)` to save 1 line on these, if desired.